### DISCUSSION TITLE
could not find `prelude` in `heron`

### DISCUSSION THREAD

**Discussion by Eldolfin** [jcornaz/heron#305] [Q&A] [answered]
I'm getting this error using heron v4.0.0-alpha.4 with bevy 0.8.0:
I'm stuck

```
error[E0432]: unresolved import `heron::prelude`
 --> src/main.rs:4:12
  |
4 | use heron::prelude::*;
  |            ^^^^^^^ could not find `prelude` in `heron`
```

**Comment by jcornaz** [comment, marked as answer]
Make sure to add either `features = ["2d"]` or `features = ["3d"]` in the dependency declaration. Example:

```toml
heron = { version = "=4.0.0-alpha.4", features = ["3d"] }
```

I am aware that it is not super intuitive. #249 aims to improve the situation.

**Reply by Eldolfin** [reply]
Thanks
